lib/linux_admin/network_interface/network_interface_rh.rb

```diff
@@ -12,6 +12,13 @@ class NetworkInterfaceRH < NetworkInterface
     def initialize(interface)
       @interface_file = self.class.path_to_interface_config_file(interface)
       super
```

Comment:
NOTE the base class initialize calls `reload` so we don't need to super then parse_conf here.